assets/javascripts/discourse/models/chat-channel.js

```diff
@@ -64,6 +64,17 @@ const ChatChannel = RestModel.extend({
     return !READONLY_STATUSES.includes(this.status);
   },
 
+  updateMembership(following, membership) {
+    this.setProperties({
+      following,
+      expanded: following,
```

Comment:
what is this property doing already?

Reply:
I have no idea it's kind of a weird one, I can't find it anywhere. Maybe we just trash it and see, the only thing I can see with expanded are chat-message and also the topic-chat-float which obviously does not rely on this

Reply:
Yes I feel this is some odd thing which shouldn't be here, I don't what an expanded membership/channel is, let's trash this.